### ISSUE TITLE
Comparison with QMIX

### ISSUE THREAD

**Issue by Jarvis-K** [hahayonghuming/VDACs#4]
I run the VmixA2C and Qmix in 3s5z, the win rate in Qmix converged to 0.8 while VmixA2C converge to 0.62.  This is not identical to the paper.

**Comment by hahayonghuming** [comment]
Hi, I changed the hyperparameters for VmixA2C recently. Specifically, vf_coef=0.1 in the current implementation whereas the previous one is 0.5. Please make sure that your configuration is consistent with mine.
And please bear in mind that all the experiments are ran independently for 5 times. I am pretty sure that there are variations between each run.

**Comment by hahayonghuming** [comment]
Also we are comparing VmixA2C with Qmix A2C version. That is Qmix is also using parallel runner where 8 episodes are ran independently. The original Qmix which is using episode runner achieves good performance (at a cost of training time). In our paper, we are comparing our algorithm with Qmix in the setting of parallel runner.

**Comment by hahayonghuming** [comment]
In our implementatin, qmix is the orignal qmix algorithm and qmix_beta is the qmix with A2C training paradigm. Make sure you are running qmix_beta as well.

**Comment by hijkzzz** [comment]
> In our implementatin, qmix is the orignal qmix algorithm and qmix_beta is the qmix with A2C training paradigm. Make sure you are running qmix_beta as well.

Why you compare with the beta version? It seems pointless to compare a worse version.

**Comment by hahayonghuming** [comment]
Let me walk you through the motivation of this study. 1, qmix is good but it is too slow; 2, we tried qmix with a2c (which we called qmix_beta, but does not mean it is beta version), It is faster than qmix but its performance drops; 3, find an alternative algorithm that offers reasonable trade-off between performance and running time; 4, compared with qmix a2x, the new algorithm does provide the trade-off.

**Comment by hijkzzz** [comment]
> Let me walk you through the motivation of this study. 1, qmix is good but it is too slow; 2, we tried qmix with a2c (which we called qmix_beta, but does not mean it is beta version), It is faster than qmix but its performance drops; 3, find an alternative algorithm that offers reasonable trade-off between performance and running time; 4, compared with qmix a2x, the new algorithm does provide the trade-off.

OK, but both CentralV and LIIR are faster than QMIX under A2C training......

![image](https://user-images.githubusercontent.com/19810594/92421458-29ba5380-f1ab-11ea-808d-ec86e7640ca5.png)

**Comment by hahayonghuming** [comment]
Yes, then we proposed other algorithms that are also faster than QMIX under A2C training and report good performance. Does this make sense?

**Comment by hijkzzz** [comment]
> Yes, then we proposed other algorithms that are also faster than QMIX under A2C training and report good performance. Does this make sense?

That's just because the default exploration ability of Actor Critic is better under the setting of A2C.
In other words, the sample diversity of Actor Critic is greater when explored in parallel.

**Comment by hahayonghuming** [comment]
Yes. You basically answered the question yourself. Because of the default exploration ability of Actor Critics under the setting of A2C, we are interested in it. QMIX has the advantage of high performance while Actor Critics has the advantage of compatibility to A2C.
